README.md

```diff
@@ -103,10 +103,14 @@ More information: [The Dangers of Square Bracket Notation](docs/the-dangers-of-s
 
 Detects insecure comparisons (`==`, `!=`, `!==` and `===`), which check input sequentially.
 
-More information: https://snyk.io/blog/node-js-timing-attack-ccc-ctf/
+More information: [Snyk Blogpost; possbily flawed.](https://snyk.io/blog/node-js-timing-attack-ccc-ctf/)
```

Comment:
We should decide what to do here. If it’s flawed and should be mentioned, it should be removed. If it’s worth mentioning, then “possibly flawed” doesn’t give enough information about how to handle the content.

Reply:
I agree. I will bring it to the security faculty, and get some opinions on these sources, and get back to this link.